debian/rules

```diff
@@ -1,33 +1,70 @@
 #!/usr/bin/make -f
+#include /usr/share/dpkg/default.mk
+#include /usr/share/dh-php/pkg-pecl.mk
 
 DH_VERBOSE = 1
 
-DPKG_EXPORT_BUILDFLAGS = 1
-include /usr/share/dpkg/default.mk
+# already in pkg-pecl.mk
+#DPKG_EXPORT_BUILDFLAGS = 1
 
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+# already in pkg-pecl.mk
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
-%:
-	dh $@ 
+# package name should be snuffleupagus, not php-snuffleupagus
```

Comment:
The package name can be `php-snuffleupagus` if you want.

Reply:
/usr/bin/dh-php only warns us about it:
    if ($#{$PACKAGE_TYPE{'has_a_module'}} >= 0)
    {
        if ($package !~ m/php-\w+?/)
        {
            warning("Package $package appears to be an PHP module. It should comply to the package naming scheme php-<modulename>\n");
        }

tideways PHP extension doesn't follow naming scheme https://salsa.debian.org/php-team/pecl/tideways

I copied pasted what I needed from the debian/rules of tideways to the debian/rules of snuffleupagus.

I wanted to avoid to generate a package.xml (which is required to build a PECL package) because I wasn't able to do it automatically.

We may reconsider this if we find it easier to generate/update the package.xml.
The prefix 'php-' and the package.xml make the debian packaging much easier as the debian/rules would contain very few lines (basically : you include pkg-pecl.mk and you export the hardening var, that's it)

Reply:
I don't get your points:

- if it's only a Debian warning, we can acknowledge and ignore it.
- if you don't want to generate a `package.xml` file, no one is forcing you to do so
- if it's easier to package for Debian with a `package.xml` file and you have issues doing it on your own, please open an issue so we can track/discuss it.